Vizzuality/climate-watch
# Feature/png modal

## PNG download modal
### Description
This PR creates a modal that displays the chart that will be later converted to `png` and downloaded. The conversion to `png` will be handled in another PR (as explained in this [Pivotal ticket](https://www.pivotaltracker.com/story/show/171697654))

![image](https://user-images.githubusercontent.com/6906348/80381582-6b53fd00-88a1-11ea-8428-29938eff2cc2.png)

### Designs
[InVision](https://projects.invisionapp.com/d/main#/console/18371781/381684669/preview)
### Testing instructions
Go to `localhost:3000//ghg-emissions` and check the `download .png` modal by clicking on the download data and save as image.
 
![image](https://user-images.githubusercontent.com/6906348/80380857-6b073200-88a0-11ea-97dd-29e03f272474.png)

### Feature relevant tickets
[Pivotal ticket](https://www.pivotaltracker.com/story/show/171785471)
